### ISSUE TITLE
What to use for current GAMMA version install?

### ISSUE THREAD

**Issue by MichaelHochriegl** [Mord3rca/gamma-launcher#94]
Greetings,

thanks so much for this launcher! Been trying to get GAMMA running on Linux for ages with the Grok Launcher and failed no matter what I tried.

With this python based launcher here I get it up and running, but I'm currently missing a couple of mods according to the GAMMA Discord. I have 442 mods after install instead of the currently included 446.

I used the `v1.5` and saw that you did a couple of commits since this release. Is there a new version bump imminent or should I use the source setup for latest GAMMA install?

**Comment by Mord3rca** [comment]
Hi,

please use installation from source, it should install all 446 mods

**Comment by MichaelHochriegl** [comment]
Thanks for the fast reply, will try that and report back once it ran through.

**Comment by Mord3rca** [comment]
You may want to follow #92 after the install to fix some definitions errors

**Comment by riku8405** [comment]
I am on 442 after using installation from source

**Comment by stubkan** [comment]
Same, 442 mods, after installing using gamma launcher only.

There may be a version mismatch?

on the discord, there are several people in the last few days saying they only have 442 mods not 446 and they all are told to update the mod launcher to version 6.0.0.0

**Comment by riku8405** [comment]
I followed steps from issue #92 as well, I can make it to the main menu, and create a new game, but before I load in it crashes

**Comment by MichaelHochriegl** [comment]
Sorry for my late reply, but it took a while till I had the chance to start the download again. For me it correctly pulled all the 446 mods when I used the main branch source and did the steps from #92 

I tried 30 minutes around with debug mode to check if the basic stuff looks and works correctly. No problems on my end. I started yesterday to alter my install with a couple of different mods and it still works fine.
